recipe/meta.yaml

```diff
@@ -47,7 +47,7 @@ requirements:
     # These are mostly run-like, but for the same reasons as
     # above, we need most of these at build time, so they are mostly
     # host-like
-    - boost =1.75
+    - boost =1.74
```

Comment:
@brianv0 Is this ok?

Reply:
Sorry @ktlim! I just remember Brian told me the stack would not build against 1.74 due to a bug.

Reply:
We have tested extensively. https://jira.lsstcorp.org/browse/DM-27301 tracks his work.  (Incidentally, this PR is implementing https://jira.lsstcorp.org/browse/RFC-761.)

Reply:
yeah bugs were addressed

Reply:
Ohhhh Sorry! This looks great!